internal/commands/extension_package.go

```diff
@@ -45,7 +45,7 @@ func ExtensionPackage(logger logging.Logger, cfg config.Config, packager Extensi
 				stringPolicy = cfg.PullPolicy
 			}
 
-			pullPolicy, err := image.ParsePullPolicy(stringPolicy)
+			pullPolicy, err := imagePullPolicyHandler.ParsePullPolicy(stringPolicy, logger)
```

Comment:
I think the `imagePullPolicyHandler` already holds a logger instance, we can remove the `logger` parameter from this method.